test/integration/kubernetes_deploy_test.rb

```diff
@@ -11,7 +11,7 @@ def test_full_hello_cloud_set_deploy_succeeds
       "Deploying ConfigMap/hello-cloud-configmap-data (timeout: 30s)",
       "Hello from Docker!", # unmanaged pod logs
       "Result: SUCCESS",
-      "Successfully deployed 12 resources"
+      "Successfully deployed 13 resources"
```

Comment:
Can you please also add an assertion about this resource to [HelloCloud#assert_all_up](https://github.com/Shopify/kubernetes-deploy/blob/master/test/helpers/fixture_sets/hello_cloud.rb#L9-L18)?